docs/releases/versioning-policy.md

```diff
@@ -1 +1,71 @@
+# Kubeflow Pipelines Versioning
+
+The Kubeflow Pipelines versioning follows the [semantic versioning policy](https://semver.org/). The Kubeflow Pipelines versions are in the format of X.Y.Z, where X is the major version, Y is the minor version, and Z is the patch version. The major version is incremented for API/feature changes that are new and not backward-compatible; the minor version is incremented for API/feature changes that are new and backward-compatible; and the patch version is for bug fixes between two minor versions. The Kubeflow Pipelines X.Y.Z refers to the version (git tag) of the released Kubeflow Pipelines and it versions all servers and the SDK of the Kubeflow Pipelines. Moreover, if the version number includes an appendix -rcN, where N is a number, the appendix indicates a release candidate, which is a pre-release version of an upcoming release.
+
+# Kubeflow Pipelines Feature Status
+
+The features in Kubeflow Pipelines fall into three different phases: general availability,
+alpha and beta. The features in the first phase are stable and have
+active support for bug fixes. The features in the latter two phases are both experimental.
+The features in alpha phase are mostly stable. On a case-by-case basis, they will either
+stay in the maintenance mode or be improved to general availability in the future. The features
+in beta phase are complete but haven't been tested extensively. They are subject to
+future deprecation.
+
+## Core features in general availability
+
+### APIs
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Pipeline Version | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Run              | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |        |         | - [x]     | - [x] |
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
```

Comment:
Can you think of a structure that contains such a table, but this table is not in GA section? because Job is not in GA

Reply:
Looks like you have job below, so maybe you forgot to delete this line?

Reply:
forgot to remove it here. removed.